zone/questmgr.cpp

```diff
@@ -2441,6 +2441,15 @@ bool QuestManager::istaskappropriate(int task) {
 	return false;
 }
 
+std::string QuestManager::gettaskname(uint32 task_id) {
+	QuestManagerCurrentQuestVars();
+
+	if (RuleB(TaskSystem, EnableTaskSystem))
```

Comment:
Please just enclose if blocks with brackets; this has created issues in the past so I'd prefer we don't use them